### DISCUSSION TITLE
Identifier and Action

### DISCUSSION THREAD

**Discussion by TienDat2k** [aieuo/Mineflow#79] [Q&A]
Please allow me to ask a few questions. - What is "identifier" and how to use it? - How can I make the entity action to attack the player when the entity touches them. I've searched for all types of events and actions but none of them match. please help me

**Comment by aieuo** [comment]
> - What is "identifier" and how to use it?

You mean Trigger identifier? If yes, you can input anything you want.  It will be used to distinguish custom triggers.

> - How can I make the entity action to attack the player when the entity touches them. I've searched for all types of events and actions but none of them match. please help me

To check if an entity is colliding with a player, you can use  `World > Get the distance between two positions`, `Script > if`, and `Script > Compare the numbers` actions.  
To attack a player, you can use `Entity > Attack the entity` action.
